src/modules/Client/Api/Guest.php

```diff
@@ -138,10 +138,13 @@ public function login($data)
 
         $service = $this->getService();
         $client = $service->authorizeClient($data['email'], $data['password']);
+        
+        if (!$service->clientAlreadyExists($data['email'])) {
+            throw new \Box_Exception('Email not found in client database.', [], 401);
```

Comment:
Errors should be the same as line 147

Just to prevent users to "discover" email addresses by quessing